applications/FemToDemApplication/custom_strategies/femdem_residual_criteria.h

```diff
@@ -159,6 +162,10 @@ class FemDemResidualCriteria
         const TSystemVectorType& rb
         ) override
     {
+        if (rModelPart.GetProcessInfo()[NL_ITERATION_NUMBER] == 1) {
+            KRATOS_INFO("") << "___________________________________________________________________" << std::endl;
+            KRATOS_INFO("") << "|    ITER     |     RATIO      |    ABS_NORM    |    CONVERGED    |" << std::endl;
```

Comment:
nice table

Reply:
I want to implement the very same in the core... Would you agree wirth something similar?

Reply:
I would use the logger_table or an improved version of it. This is not the best way to do it

Reply:
And use colors :D